src/modules/rc_update/rc_update.cpp

```diff
@@ -78,8 +106,46 @@ RCUpdate::init()
 void
 RCUpdate::parameters_updated()
 {
-	/* read the parameter values into _parameters */
-	update_parameters(_parameter_handles, _parameters);
+	if (_param_rc_flt_smp_rate.get() < 1.0f) {
+		_param_rc_flt_smp_rate.set(1.0f);
+		_param_rc_flt_smp_rate.commit_no_notification();
+	}
+
+	// make sure the filter is in its stable region -> fc < fs/2
+	const float flt_cutoff_min = _param_rc_flt_smp_rate.get() / 2.0f - 1.0f;
+
+	if (_param_rc_flt_cutoff.get() < flt_cutoff_min) {
```

Comment:
It's an upper bound:
```suggestion
	if (_param_rc_flt_cutoff.get() > flt_cutoff_max) {
```

Reply:
Oops, nice catch.